common/defs-flags.go

```diff
@@ -1,134 +1,134 @@
+	FlagWorkflowIDTerminate        = "Terminate Workflow Execution by Id."
+	FlagQueryTerminate             = "Terminate Workflow Executions by List Filter. See https://docs.temporal.io/concepts/what-is-a-list-filter/."
+	FlagEventIDDefinition          = "The Event Id for any Event after WorkflowTaskStarted you want to reset to (exclusive). It can be WorkflowTaskCompleted, WorkflowTaskFailed or others."
+	FlagQueryResetBatch            = "Visibility Query of Search Attributes describing the Workflow Executions to reset. See https://docs.temporal.io/docs/tctl/workflow/list#--query."
+	FlagInputFileReset             = "Input file that specifies Workflow Executions to reset. Each line contains one Workflow Id as the base Run and, optionally, a Run Id."
+	FlagExcludeFileDefinition      = "Input file that specifies Workflow Executions to exclude from resetting."
+	FlagInputSeparatorDefinition   = "Separator for the input file. The default is a tab (\t)."
+	FlagParallelismDefinition      = "Number of goroutines to run in parallel. Each goroutine processes one line for every second."
+	FlagSkipCurrentOpenDefinition  = "Skip a Workflow Execution if the current Run is open for the same Workflow Id as the base Run."
+	FlagSkipBaseDefinition         = "Skip a Workflow Execution if the base Run is not the current Run."
 	FlagNonDeterministicDefinition = "Reset Workflow Execution only if its last Event is WorkflowTaskFailed with a nondeterministic error."
-	FlagDryRunDefinition = "Simulate reset without resetting any Workflow Executions."
-	FlagDepthDefinition = "Number of Child Workflows to expand, -1 to expand all Child Workflows."
-	FlagConcurrencyDefinition = "Request concurrency."
-	FlagNoFoldDefinition = "Disable folding. All Child Workflows within the set depth will be fetched and displayed."
-
+	FlagDryRunDefinition           = "Simulate reset without resetting any Workflow Executions."
+	FlagDepthDefinition            = "Number of Child Workflows to expand, -1 to expand all Child Workflows."
+	FlagConcurrencyDefinition      = "Request concurrency."
+	FlagNoFoldDefinition           = "Disable folding. All Child Workflows within the set depth will be fetched and displayed."
 
 	// Stack trace query flag definitions
-	FlagInputSTQDefinition = "Optional Query input, in JSON format.\nFor multiple parameters, concatenate them and separate by space."
-	FlagInputFileSTQDefinition = "Passes optional Query input from a JSON file.\nIf there are multiple JSON, concatenate them and separate by space or newline.\n" + "Input from the command line will overwrite file input."
+	FlagInputSTQDefinition             = "Optional Query input, in JSON format.\nFor multiple parameters, concatenate them and separate by space."
+	FlagInputFileSTQDefinition         = "Passes optional Query input from a JSON file.\nIf there are multiple JSON, concatenate them and separate by space or newline.\n" + "Input from the command line will overwrite file input."
```

Comment:
nit: why aren't we using multiline strings here for better readability?

Reply:
I'll change in a new PR

cc @DJSanti